.github/workflows/manage-pending-labels.yml

```
@@ -0,0 +1,42 @@
+# then those labels will be removed, providing it was not user khulnasoft who added the commend. 
+name: 🎯 Add/ Remove Awaiting Response Labels
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  remove-stale:
+   runs-on: ubuntu-latest
+   if: ${{ github.event.comment.author_association != 'COLLABORATOR' && github.event.comment.author_association != 'OWNER' }}
+   steps:
+   - name: Remove Stale labels when Updated
+     uses: actions-cool/issues-helper@v2
```

actionlint

the runner of "actions-cool/issues-helper@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)